scripts/ci-test.sh

```diff
@@ -25,4 +25,4 @@ cd "${REPO_ROOT}" && \
 	source ./scripts/fetch_ext_bins.sh && \
 	fetch_tools && \
 	setup_envs && \
-	make test
+	make test-go
```

Comment:
Are we testing generate/lint elsewhere?

Reply:
Yes, `pull-cluster-api-make` runs `lint-full docker-build` targets, which covers these already